content/conda-environment.md

```diff
@@ -2,20 +2,14 @@
 
 # Creating a Conda environment for CodeRefinery workshops
 
-1. Make a new folder for a workshop at somewhere you can easily access (e.g.
-   ./coderefinery-workshop).
-2. Download
-   [environment.yml](https://raw.githubusercontent.com/coderefinery/software/main/environment.yml)
-   from [CodeRefinery's software
-   repository](https://github.com/coderefinery/software/blob/main/environment.yml)
-   or make an `environment.yml` with the same contents inside the newly made
-   folder.
-3. In Bash, navigate to the folder where you put the `environment.yml`
-   (the file names the environment `coderefinery` automatically).
-4. If you have not, activate `conda` in Miniconda first using `conda activate` and if this does not
+1. Open your terminal shell (e.g. Bash)
+2. If you have not, activate `conda` in Miniconda first using `conda activate` and if this does not
    work, please first follow {ref}`setting-conda-path`.
-5. Run the following command: `conda env create -f environment.yml`
-6. Make sure that you have "coderefinery" folder in Miniconda3/envs/. If you
+3. Run the following command:
+   ```
+   conda env create -f https://raw.githubusercontent.com/coderefinery/software/main/environment.yml
+   ```
+4. Make sure that you have "coderefinery" folder in Miniconda3/envs/. If you
    use Anaconda3, then it should be in Anaconda3/envs/.
```

Comment:
Is it better to `conda env list` ?  Then we don't have to expect people to know where these directories are.

Reply:
I agree. Adjusting in a follow-up commit.